simple-service-webapp-tests/src/test/resources/credentials.properties

```diff
@@ -1,3 +1,3 @@
-url=url_here
-username=username_here
-password=password_here
\ No newline at end of file
+url=http://localhost:8080/api/authentication
```

Comment:
When we named the file, I made a mistake, it was actually meant to be called connection.properties.
It's because, the url_here thing was meant to serve as a base url for all tests. It shouldn't be just for /authentication. The idea is that if you have multiple version of your rest api deployed in the future, by simply modifying your file. All your tests will run on that version. So the url should be the base, http://localhost:8080/api/, and all tests should use it and concatenate the path ( /authentication, /userstories etc..)